src/styles/main.scss

```
@@ -10,5 +10,6 @@ body {
   box-sizing: border-box;
 }
 
-@import "_header.scss";
-@import "_catalog.scss";
+@import "./utils/_variables.scss";
```

It would be good if you signed the name of the variables file in the same style as the files in styles 

```suggestion
@import "./utils/__variables.scss";
```